product/views/ProvisionCloudTemplates.yaml

```diff
@@ -0,0 +1,104 @@
+#
+# This is an MIQ Report configuration file
+#   Single value parameters are specified as:
+#     single_value_parm: value
+#   Multiple value parameters are specified as:
+#     multi_value_parm:
+#       - value 1
+#       - value 2
+#
+
+# Report title
+title: Images
+
+# Menu name
+name: Images
+
+# Main DB table report is based on
+db: TemplateCloud
```

Comment:
`TemplateCloud` is right if the other is using `TemplateInfra`,
otherwise this should be `ManageIQ::Providers::CloudManager::Template`

(Template* is an alias to ManageIQ::Providers::*Manager::Template)